assets/src/edit-story/components/library/panes/text/textTab.js

```diff
@@ -82,8 +81,7 @@ function TextTab(props) {
   const handleAddText = (evt) => {
     evt.stopPropagation();
     insertElement('text', {
-      content: __('Fill in some text', 'web-stories'),
-      fontSize: DEFAULT_FONT_SIZE,
+      ...DEFAULT_PRESET,
```

Comment:
Would be ideal to have a test for `textTab` which would check that on clicking `QuickAction`, `insertElement` is called with the correct attributes.

Reply:
Added a new test. PTAL